pantheon-bundle/src/main/java/com/redhat/pantheon/model/Rendering.java

```diff
@@ -0,0 +1,23 @@
+package com.redhat.pantheon.model;
+
+import com.redhat.pantheon.model.api.Field;
+import com.redhat.pantheon.model.api.SlingModel;
+import com.redhat.pantheon.servlet.AssemblyRendering;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.SlingHttpServletResponse;
+import org.apache.sling.models.annotations.Default;
+import org.osgi.framework.Constants;
+import org.osgi.service.component.annotations.Component;
+
+import javax.inject.Named;
+import java.io.IOException;
+import java.util.Calendar;
+
+/**
+ * Represents rendering as an interface to be implemented.
+ *
+ */
+
+public interface Rendering {
```

Comment:
I like this idea, but I would move it to the `servlet` package

Reply:
Also consider renaming it to "DocumentRenderer" - class/interface names that end in "-ing" are unusual, and adding "Document" to the front of it makes it a bit more specific.

Reply:
Or... now that I think about it... maybe something more like "DocumentServletHandler" since this doesn't just render, but also takes care of populating the servlet response as well. That would make the name even more targeted toward its actual function.

Reply:
@benradey  This is an interface, we can name implementation class as handler, but renaming it to DocumentRedererHandler doesn't sound good.

Reply:
Deprecated.